common/views.py

```diff
@@ -28,15 +30,25 @@ def form_invalid(self, form):
 
 
 class AccountSignup(FormView):
-    form_class = UserCreationForm
+    form_class = RegisterForm
     template_name = 'common/signup.html'
     success_url = '/home'
```

Comment:
why success_url if you are not using it,
use this success_url instead of HttpResponseRedirect in form_valid return